driver-sync/src/test/functional/com/mongodb/client/AbstractUnifiedTest.java

```diff
@@ -305,12 +318,20 @@ public void shouldPassAllOutcomes() {
             List<CommandEvent> expectedEvents = getExpectedEvents(definition.getArray("expectations"), databaseName, null);
             List<CommandEvent> events = commandListener.getCommandStartedEvents();
 
-            assertEventsEquality(expectedEvents, events, lsidMap);
+            assertEventsEquality(expectedEvents, events.subList(0, expectedEvents.size()), lsidMap);
         }
```

Comment:
Why is a sublist used here? Is this a change that should be pushed down to the assertion method?

Reply:
There are CRUD operations in the new JSON tests that generate an extra event. Sublists are also used in `AbstractCrudTest`.